src/right-hand-panel/use-right-hand-panel-context.js

```diff
@@ -0,0 +1,6 @@
+import { useContext } from 'react'
+import RightHandPanelContext from './right-hand-panel-context.js'
+
+export default function useRightHandPanelContext() {
```

Comment:
Let's create some hooks to just expose the `set`-function. It seems like this is used in every `InputField`, and thus all field's are re-rendered whenever the panel is shown. That is not necessary.


```javascript
export function useShowRightHandPanel() {
    const { show } = useContext(RightHandPanelContext)
    return useMemo((id) => {
        show(id)
    }, [show])
}

export function useHideRightHandPanel() {
    const { hide } = useContext(RightHandPanelContext)
    return useMemo(() => {
        hide()
    }, [hide])
}
```